camviewer_ui_impl.py

```diff
@@ -2470,13 +2499,44 @@ def dumpConfig(self):
                     % (i + 1, lMarker[i].abs().x(), lMarker[i].abs().y())
                 )
             g.write("dispspec    " + str(self.dispspec) + "\n")
+            f.write(
```

Comment:
This config thing is some weird format made up for camviewer, right? This thing reads it?
https://github.com/mcb64/camviewer/blob/0bf97cb79cf1476e06b935b7274b5a129b1b750f/camviewer_ui_impl.py#L78-L103

What's the difference between an oldcfg and a newcfg?
Am I understanding it correctly that it'll load the old format, write the new format, and then delete the old one?

Reply:
Yes.  I keep state files for each camera you view in ~/.yagviewer/BLAH.  So it will remember your orientation, your marker settings, etc. etc.

Originally back when dinosaurs roamed the earth, this was just the PV base name.  But as camera types proliferated, I decided that this should be PVNAME,TYPE, so if a camera gets upgraded or otherwise changed, the old state is ignored.  Now, that we've decided to drop everything except area detector, I'm back to just PV base name.

However, I think I have told you that I am an evangelist for the First Church of Backwards Compatibility... so it would be heresy to just ignore the existing PVNAME,GE files!  So this code tries to find an old one, read it in... and delete it, so the user sins no more...

It's the same format, just a different name.